panther_manager/test/plugins/test_call_set_bool_service_node.cpp

```diff
@@ -0,0 +1,163 @@
+{
+  std::map<std::string, std::string> service = {{"service_name", "set_bool"}, {"data", "true"}};
+
+  panther_manager_plugin_test::PantherManagerPluginTestUtils test_utils;
+  test_utils.Start();
+  EXPECT_THROW({ test_utils.CreateTree("WrongCallSetBoolService", service); }, BT::RuntimeError);
+  test_utils.Stop();
+}
+
+TEST(TestCallSetBoolService, wrong_call_set_bool_service_service_server_not_initialized)
+{
+  std::map<std::string, std::string> service = {{"service_name", "set_bool"}, {"data", "true"}};
+
+  panther_manager_plugin_test::PantherManagerPluginTestUtils test_utils;
+  test_utils.Start();
+  auto & tree = test_utils.CreateTree("CallSetBoolService", service);
+
+  auto status = tree.tickWhileRunning(std::chrono::milliseconds(100));
+  EXPECT_EQ(status, BT::NodeStatus::FAILURE);
+
+  test_utils.Stop();
+}
+
+TEST(TestCallSetBoolService, good_set_bool_call_service_success_with_true_value)
+{
+  std::map<std::string, std::string> service = {{"service_name", "set_bool"}, {"data", "true"}};
+
+  panther_manager_plugin_test::PantherManagerPluginTestUtils test_utils;
+  test_utils.Start();
+  auto & tree = test_utils.CreateTree("CallSetBoolService", service);
+
+  test_utils.CreateSetBoolServiceServer(ServiceSuccessCallbackCheckTrueValue);
+
+  auto status = tree.tickWhileRunning(std::chrono::milliseconds(100));
+  EXPECT_EQ(status, BT::NodeStatus::SUCCESS);
+
+  test_utils.Stop();
+}
+
+TEST(TestCallSetBoolService, good_set_bool_call_service_success_with_false_value)
+{
+  std::map<std::string, std::string> service = {{"service_name", "set_bool"}, {"data", "false"}};
+
+  panther_manager_plugin_test::PantherManagerPluginTestUtils test_utils;
+  test_utils.Start();
+  auto & tree = test_utils.CreateTree("CallSetBoolService", service);
+
+  test_utils.CreateSetBoolServiceServer(ServiceSuccessCallbackCheckFalseValue);
+
+  auto status = tree.tickWhileRunning(std::chrono::milliseconds(100));
+  EXPECT_EQ(status, BT::NodeStatus::SUCCESS);
+
+  test_utils.Stop();
+}
+
+TEST(TestCallSetBoolService, wrong_set_bool_call_service_failure)
+{
+  std::map<std::string, std::string> service = {{"service_name", "set_bool"}, {"data", "false"}};
+
+  panther_manager_plugin_test::PantherManagerPluginTestUtils test_utils;
+  test_utils.Start();
+  auto & tree = test_utils.CreateTree("CallSetBoolService", service);
+
+  test_utils.CreateSetBoolServiceServer(ServiceFailedCallback);
+
+  auto status = tree.tickWhileRunning(std::chrono::milliseconds(100));
+  EXPECT_EQ(status, BT::NodeStatus::FAILURE);
+
+  test_utils.Stop();
+}
+
+TEST(TestCallSetBoolService, wrong_service_value_defined)
+{
+  std::map<std::string, std::string> service = {
+    {"service_name", "set_bool"}, {"data", "wrong_bool"}};
+
+  panther_manager_plugin_test::PantherManagerPluginTestUtils test_utils;
+  test_utils.Start();
+  auto & tree = test_utils.CreateTree("CallSetBoolService", service);
+
+  auto status = tree.tickWhileRunning(std::chrono::milliseconds(100));
```

Comment:
```suggestion
  auto & tree = test_utils.CreateTree("CallSetBoolService", service);
  
  test_utils.CreateSetBoolServiceServer(ServiceSuccessCallbackCheckTrueValue);

  auto status = tree.tickWhileRunning(std::chrono::milliseconds(100));
```
You missed service initialization